src/Control/Parallel.purs

```diff
@@ -48,3 +50,13 @@ parSequence_
   => t (m a)
   -> m Unit
 parSequence_ = parTraverse_ id
+
+parOneOf
+  :: forall a t m f
+   . Parallel f m
+  => Plus f
```

Comment:
If the function doesn't take advantage of those laws, then what does it even mean?